karma/test-services.coffee

```diff
@@ -634,5 +634,22 @@ describe('services:', () ->
         $httpBackend.flush()
       )
     )
+
+    describe('trigger', () ->
+      it('triggers an action', () ->
+        $httpBackend.expectPUT('/pods/action/21/', {
+            data: {
+              type: 'foo'
```

Comment:
I expected there to be a comma after `type: 'foo'`. I'm guessing it's not necessary but if we can have it, it might be nice to have?